src/utils/IdealGraphVisualizer/Graph/src/main/java/com/sun/hotspot/igv/graph/FigureConnection.java

```diff
@@ -105,6 +105,7 @@ public String getToolTipText() {
         builder.append(getOutputSlot().getFigure().getProperties().resolveString(shortNodeText));
         builder.append(" → ");
         builder.append(getInputSlot().getFigure().getProperties().resolveString(shortNodeText));
+        builder.append(" [" + getInputSlot().getPosition() + "]");
```

Comment:
String concatenation could be replaced by `append()` calls to follow the pattern above (the `builder.` can also be removed and the calls be chained together directly):
```suggestion
        builder.append(" [")
               .append(getInputSlot().getPosition())
               .append("]");
```